src/main/java/mate/academy/spring/controller/InitialDataController.java

```diff
@@ -0,0 +1,98 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.NoSuchElementException;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.Movie;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.MovieService;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/inject")
+public class InitialDataController {
+    private final CinemaHallService cinemaHallService;
+    private final MovieService movieService;
+    private final MovieSessionService movieSessionService;
+    private final ShoppingCartService shoppingCartService;
+    private final UserService userService;
+    private final OrderService orderService;
+
+    public InitialDataController(CinemaHallService cinemaHallService,
+                                 MovieService movieService,
+                                 MovieSessionService movieSessionService,
+                                 ShoppingCartService shoppingCartService,
+                                 UserService userService,
+                                 OrderService orderService) {
+        this.cinemaHallService = cinemaHallService;
+        this.movieService = movieService;
+        this.movieSessionService = movieSessionService;
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.orderService = orderService;
+    }
+
+    @GetMapping
```

Comment:
The method injectInitialData should be public instead of private, as it is a controller method exposed through a GET request.